Resources/Prototypes/_Mono/Entities/Clothing/Belt/belts.yml

```
@@ -0,0 +1,27 @@
+    size: Ginormous
+  - type: Storage
+    maxItemSize: Large
+    grid:
+      - 0,0,6,2
+
+- type: entity
+  parent: ClothingBeltMilitaryLeskala
+  id: ClothingBeltMilitaryLeskalaFilled
+  suffix: Filled
+  components:
+  - type: StorageFill
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)